components/Dashboard/FormEdit.tsx

```diff
@@ -131,7 +131,7 @@ const FormEdit: React.FC<FormEditProps> = ({ reviewForm, onStepChange }) => {
                 <div className="relative mt-4 w-fit">
                   <input
                     disabled={form.formState.isSubmitting}
-                    className="w-[7.5rem] rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
+                    className="w-auto rounded bg-transparent px-1 py-1 text-xl font-bold outline-dashed outline-1 outline-transparent duration-100 hover:outline-gray-300 focus:outline-gray-300"
```

Comment:
This can be dangerous, We need to put a max-width as very long names can eventually overflow.
CC @hemantwasthere